parkings/models/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .addresses import *
+from .operators import *
+from .parkings import *
```

Comment:
Yeah, we seem to have standardized on having the app name in plural and these kinds of break-out files in singular.

Also, according to the Zen of Python, _explicit is better than implicit_, so import the classes you need explicitly instead of `*`. This saves some time when someone tries to figure out where the classes are actually defined.